### ISSUE TITLE
Dragging SC text update

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#1633]
Just posting this from an email for tracking purposes. @mbgower wrote:
> To me, a single word alteration may avoid a lot of potential confusion. I realize that we are just mirroring language in 2.5.1; however, for whatever reason, it seems more easily misinterpreted here.
> I'm just worried about someone failing 'swipe left/right' options (to delete or reply) on an email inbox because there is not a way of operating the swipes without dragging; whereas the function can be achieved by tapping the email message and then deleting it by tapping a delete or reply buttons on the subsequent page.

The suggestion is to replace 'operated' with 'achieved'.

All functionality that uses a <a>dragging movement</a> for operation can be **achieved** by a single pointer without dragging, unless dragging is <a>essential

**Comment by detlevhfischer** [comment]
Could we add the proposed ‘single word alteration’ here, to make this easier to process?

**Comment by alastc** [comment]
Sorry @detlevhfischer, I've added that above. It was also in the survey.
